pkg/util/ipvs/ipvs.go

```diff
@@ -43,7 +43,7 @@ type Interface interface {
 	DeleteRealServer(*VirtualServer, *RealServer) error
 }
 
-// VirtualServer is an user-oriented definition of an IPVS virtual server in its entirety.
+// VirtualServer is an user-oriented definition of an IPVS Server in its entirety.
```

Comment:
Tere are two "server" in IPVS: virtual server and real server. So, IPVS Server will make people wonder what the "server" is?

Reply:
Makes sense, changed back.